src/include/wiredtiger.in

```diff
@@ -1659,6 +1659,22 @@ struct __wt_session {
 	 */
 	int __F(prepare_transaction)(WT_SESSION *session, const char *config);
 
+	/*!
+	 * Reset the snapshot.
+	 *
+	 * This method resets snapshots for snapshot isolation transactions to
```

Comment:
Same here, what are pros and cons of resetting a snapshot, what is the use of resetting a snapshot.

Reply:
Fixed